client/my-sites/plans/main.jsx

```diff
@@ -20,15 +20,18 @@ import Main from 'components/main';
 import Notice from 'components/notice';
 import observe from 'lib/mixins/data-observe';
 import paths from './paths';
-import PlanList from 'components/plans/plan-list' ;
+import OldPlanList from 'components/plans/plan-list' ;
+import PlanFeaturesList from 'my-sites/plan-features-list';
 import PlanOverview from './plan-overview';
 import { shouldFetchSitePlans, plansLink } from 'lib/plans';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import { SUBMITTING_WPCOM_REQUEST } from 'lib/store-transactions/step-types';
 import UpgradesNavigation from 'my-sites/upgrades/navigation';
 import QueryPlans from 'components/data/query-plans';
 import { PLAN_MONTHLY_PERIOD } from 'lib/plans/constants';
+import { abtest } from 'lib/abtest';
 
+const showPlanFeatures = abtest( 'planFeatures' ) === 'show';
```

Comment:
We should also check for the feature flag, since the route does this too.
`config.isEnabled( 'manage/plan-features' ) && abtest( 'planFeatures' ) === 'show' )`